Single-line multi assignment doesn't work

**V version:** V 0.1.27 8f6d69b
**OS:** Linux

**What did you do?**
Tried to compile the code:

```v
module main

fn main() {
    a, b, c, d := 1, 2, 3, 4
    println('$a, $b, $c, $d')
}

```

**What did you expect to see?**

Successful compilation.
 
**What did you see instead?**

```
==================
/home/[user]/.cache/v/main.tmp.c: In function ‘main’:
/home/[user]/.cache/v/main.tmp.c:9434:6: error: variable or field ‘mr_40’ declared void
 9434 | void mr_40 = 1;
      |      ^~~~~
/home/[user]/.cache/v/main.tmp.c:9435:14: error: request for member ‘arg0’ in something not a structure or union
 9435 | int a = mr_40.arg0;
      |              ^
/home/[user]/.cache/v/main.tmp.c:9436:14: error: request for member ‘arg1’ in something not a structure or union
 9436 | int b = mr_40.arg1;
      |              ^
/home/[user]/.cache/v/main.tmp.c:9437:14: error: request for member ‘arg2’ in something not a structure or union
 9437 | int c = mr_40.arg2;
...
==================
(Use `v -cg` to print the entire error message)

builder error: C error.

Please make sure that:
- You have all V dependencies installed.
- You did not declare a C function that was not included. (Try commenting your code that involves C interop)
- You are running the latest version of V. (Try running `v up` and rerunning your command)

If you're confident that all of the above is true, please try running V with the `-cg` option which enables more debugging capabilities.
```

--- a/vlib/v/gen/cgen.v
+++ b/vlib/v/gen/cgen.v
@@ -845,7 +845,7 @@ fn (mut g Gen) gen_assign_stmt(assign_stmt ast.AssignStmt) {
 	}
 	mut is_multi := false
 	// json_test failed w/o this check
-	if return_type != 0 {
+	if return_type != table.void_type && return_type != 0 {
 		sym := g.table.get_type_symbol(return_type)
 		// the left vs. right is ugly and should be removed
 		is_multi = sym.kind == .multi_return || assign_stmt.left.len > assign_stmt.right.len ||

--- a/vlib/v/tests/multiple_assign_test.v
+++ b/vlib/v/tests/multiple_assign_test.v
@@ -1,0 +1,6 @@
+fn test_multiple_assign() {
+	a, b, c := 1, 2, 3
+	assert a == 1
+	assert b == 2
+	assert c == 3
+}

